packages/rocketchat-message-snippet/client/tabBar/views/snippetMessage.html

```diff
@@ -0,0 +1,67 @@
+<template name="snippetMessage">
```

Comment:
I did not want to use message because I only want to display the filename of the message as a link (not the entire message with the content).So I thought creating a new template for this special message was a better idea than touch the messages template.